app/jsm.yml

```diff
@@ -48,11 +48,11 @@ execution:
     hold-for: ${test_duration}
 scenarios:
   selenium_customers:
-    script: selenium_ui/jsd_ui_customers.py
+    script: selenium_ui/jsm_ui_customers.py
   selenium_agents:
-    script: selenium_ui/jsd_ui_agents.py
+    script: selenium_ui/jsm_ui_agents.py
   locust:
-    script: locustio/jsd/locustfile.py
+    script: locustio/jsm/locustfile.py
   jmeter:
     script: jmeter/jsd.jmx
```

Comment:
jsd.jmx -> jsm.jmx ?
or your changes only for selenium ?

Reply:
I do not include JMeter naming changes to this PR.